frontend/src/lib/candidate/components/PasswordSetComponent/PasswordSetComponent.svelte

```diff
@@ -0,0 +1,77 @@
+<script lang="ts">
+  import {t} from '$lib/i18n';
+  import {Button} from '$lib/components/button';
+  import PasswordField from '../PasswordField/PasswordField.svelte';
+  import {getRoute, Route} from '$lib/utils/navigation';
+  import PasswordValidator from '../passwordValidator/PasswordValidator.svelte';
+  import {createEventDispatcher} from 'svelte';
+  export let password1 = '';
+  export let password2 = '';
+  export let autocomplete1 = '';
+  export let autocomplete2 = '';
+  export let errorMessage = '';
+  export let validPassword = false;
+
+  $: disableSetButton = validPassword && password2.length > 0;
+  //dispatcher used for a function that calls the submit button's function in parent component
+  const dispatch = createEventDispatcher();
+  export function ButtonPressed() {
+    dispatch('ButtonPressed');
+  }
+</script>
+
+<!--
+@component
+PasswordSetComponent is a component used by PasswordSetPage and PasswordResetPage, that contains a password input field,
+a second confirmation password input field, a button for submitting the password, a button to contact support and informatory text
+
+### Properties
+
+- `password1` : variable the first password input box uses for its value
+- `password2` : variable the second password input box uses for its value
+- `autoComplete1` : variable used for first field's password autocomplete value
+- `autoComplete2` : variable used for second field's password autocomplete value
+- `errorMessage` : message showing error if one exists
+- `validPassword` : variable used to determine if the password combination is valid
+
+### Usage
+
+```tsx
+
+<PasswordSetComponent
+bind:password1={passwordOfContext1}
+bind:password2={passwordOfContext2}
+autocomplete1="autocomplete1"
+autocomplete2="autocomplete2"
```

Comment:
The example could show the actual usage of the autocomplete attribute, i.e. "new-password", for example